src/components/modules/searchQueriesComparison/QueryHeaderPanel.vue

```diff
@@ -211,6 +212,11 @@ export default {
 
 <style lang="scss">
   @import "impresso-theme/src/scss/variables.sass";
+  $left-panel-color: #2e80c9;
```

Comment:
For diverging bar charts the `A` and `B` colours are defined dynamically in the page data object: https://github.com/impresso/impresso-frontend/blob/feature/inspect_compare_diverging_bar_chart/src/components/SearchQueriesComparisonPage.vue#L280-L282 . Considering how often these colours are used in this module I'm not sure how to pass them from a javascript variable into scss. Probably using `:style` on top level element is not going to be enough...